### PR TITLE
Prep proj for initial publish

### PR DESCRIPTION
**Pull request by hootio**
Followed these steps:
https://code.visualstudio.com/api/working-with-extensions/publishing-extension

### CODE REVIEW

**Review by supervanya — APPROVED** [review]
LGTM! Sorry I forgot to review 🙈

File: README.md [hunk 1]
```diff
@@ -24,8 +24,8 @@ For example:
 
 This extension contributes the following settings:
 
-* `myExtension.enable`: enable/disable this extension
-* `myExtension.thing`: set to `blah` to do something
+- `myExtension.enable`: enable/disable this extension
+- `myExtension.thing`: set to `blah` to do something
```
Comment (supervanya) [inline README.md:28]: Is this going to be shown in the extension "store" page? Maybe we can make a copy of this and remove most of the things that are placeholder?

**Review by supervanya — CHANGES_REQUESTED** [review]
Changed my mind 🙈

File: README.md (hunk 1 above)
Comment (hootio) [reply README.md:28]: Oh my, too late.

Comment (hootio) [reply README.md:28]: Let's do that next

Comment (supervanya) [reply README.md:28]: Ok I will open a new PR

**Pull request merged**